server/protocols/jmap/pom.xml

```diff
@@ -243,6 +243,12 @@
                     <type>test-jar</type>
                 </dependency>
                 <dependency>
+                    <groupId>com.jayway.jsonpath</groupId>
```

Comment:
Version & artifact may be put in parent as this artifact is shared with more than one project.